Update release notes for Elastic Cloud Serverless (#3431)
elastic/docs-content: release-notes/elastic-cloud-serverless/index.md

* Fixes the autocomplete configuration for the `pinned` retriever by removing the `match_criteria` field [#234903]({{kib-pull}}234903)
* Fixes a bug by allowing the use of `cmd + /` for comment toggling in the Monaco editor [#235334]({{kib-pull}}235334)
* Adds a check for all privileges for {{sec-serverless}} when creating lists [#234602]({{kib-pull}}234602)
* Correctly update SLM stats with master shutdown [#134152](https://github.com/elastic/elasticsearch/pull/134152)
* Bug fix: Facilitate second retrieval of the same value [#134790](https://github.com/elastic/elasticsearch/pull/134790)
* Avoid holding references to SearchExecutionContext in SourceConfirmedTextQuery [#134887](https://github.com/elastic/elasticsearch/pull/134887)
* Add exception for perform embedding inference requests with query provided [#131641](https://github.com/elastic/elasticsearch/pull/131641)

> * Fixed match only text block loader not working when a keyword multi field is present [#134582](https://github.com/elastic/elasticsearch/pull/134582)

Author: ```suggestion
* Fixes match only text block loader not working when a keyword multi field is present [#134582](https://github.com/elastic/elasticsearch/pull/134582)
```

* Fixing conditional processor mutability bugs [#134936](https://github.com/elastic/elasticsearch/pull/134936)
* [Transform] Wait for PIT to close [#134955](https://github.com/elastic/elasticsearch/pull/134955)
* Bypass MMap arena grouping as this has caused issues with too many regions being mapped [#135012](https://github.com/elastic/elasticsearch/pull/135012)

> * Fix deadlock in ThreadPoolMergeScheduler when a failing merge closes the IndexWriter [#134656](https://github.com/elastic/elasticsearch/pull/134656)

Author: ```suggestion
* Fixes deadlock in ThreadPoolMergeScheduler when a failing merge closes the IndexWriter [#134656](https://github.com/elastic/elasticsearch/pull/134656)
```

* Adds a link to Agent Builder in the **View Data** dropdown [#234679]({{kib-pull}}234679)
* Adds the AutoOps Search tier page, which provides project-level insights and deeper insights into {{serverless-short}} resources (VCUs) and performances
% Relates to https://github.com/elastic/autoops/issues/20 and https://github.com/elastic/autoops/issues/200

> * Add relevant attributes to search took time APM metrics [#134232](https://github.com/elastic/elasticsearch/pull/134232)

Reviewer: ```suggestion
* Adds relevant attributes to search took time APM metrics [#134232](https://github.com/elastic/elasticsearch/pull/134232)
```

* [ML] Adding headers support for OpenAI chat completion [#134504](https://github.com/elastic/elasticsearch/pull/134504)
* Extend kibana-system permissions to manage security entities [#133968](https://github.com/elastic/elasticsearch/pull/133968)
* Track shardStarted events for simulation in DesiredBalanceComputer [#133630](https://github.com/elastic/elasticsearch/pull/133630)
* Add file extension metadata to cache miss counter from SharedBlobCacheService [#134374](https://github.com/elastic/elasticsearch/pull/134374)
* Remove ingest conditionals _type deprecation warning [#134851](https://github.com/elastic/elasticsearch/pull/134851)

> * Allow including semantic field embeddings in _source [#134717](https://github.com/elastic/elasticsearch/pull/134717)

Reviewer: ```suggestion
* Allows including semantic field embeddings in `_source` [#134717](https://github.com/elastic/elasticsearch/pull/134717)
```

* Integrate weights into simplified RRF retriever syntax [#132680](https://github.com/elastic/elasticsearch/pull/132680)

> * DLM: Better `max_age` rollover for tiny retentions [#134941](https://github.com/elastic/elasticsearch/pull/134941)

Reviewer: ```suggestion
* Adjusts rollover criteria to have a better `max_age` rollover for tiny retentions [#134941](https://github.com/elastic/elasticsearch/pull/134941)
```

* ES|QL: add support for include_execution_metadata parameter [#134446](https://github.com/elastic/elasticsearch/pull/134446)
* ES|QL: Add telemetry support for Lookup Join On Expression [#134942](https://github.com/elastic/elasticsearch/pull/134942)

> * Improve block loader for source only runtime fields of type keyword [#135026](https://github.com/elastic/elasticsearch/pull/135026)

Reviewer: ```suggestion
* Improves block loader for source-only runtime fields of type keyword [#135026](https://github.com/elastic/elasticsearch/pull/135026)
```

* Optimize BytesArray::indexOf, which is used heavily in ndjson parsing [#135087](https://github.com/elastic/elasticsearch/pull/135087)
* Make SecureString comparisons constant time [#135053](https://github.com/elastic/elasticsearch/pull/135053)

> * ESQL: URL encoding changes [#134503](https://github.com/elastic/elasticsearch/pull/134503)

Reviewer: ```suggestion
* Updates URL encoding in {{esql}} [#134503](https://github.com/elastic/elasticsearch/pull/134503)
```

* Fixes the autocomplete configuration for the `pinned` retriever by removing the `match_criteria` field [#234903]({{kib-pull}}234903)
* Fixes a bug by allowing the use of `cmd + /` for comment toggling in the Monaco editor [#235334]({{kib-pull}}235334)
* Adds a check for all privileges for {{sec-serverless}} when creating lists [#234602]({{kib-pull}}234602)
* Correctly update SLM stats with master shutdown [#134152](https://github.com/elastic/elasticsearch/pull/134152)

> * Bug fix: Facilitate second retrieval of the same value [#134790](https://github.com/elastic/elasticsearch/pull/134790)

Reviewer: ```suggestion
* Fixes a bug to facilitate second retrieval of the same value [#134790](https://github.com/elastic/elasticsearch/pull/134790)
```

* Avoid holding references to SearchExecutionContext in SourceConfirmedTextQuery [#134887](https://github.com/elastic/elasticsearch/pull/134887)
* Adds exception for perform embedding inference requests with query provided [#131641](https://github.com/elastic/elasticsearch/pull/131641)

> * Fixes match only text block loader not working when a keyword multi field is present [#134582](https://github.com/elastic/elasticsearch/pull/134582)

Reviewer: ```suggestion
* Fixes a bug where the match only text block loader was not working correctly when a keyword multi-field was present [#134582](https://github.com/elastic/elasticsearch/pull/134582)
```

* Fixes conditional processor mutability bugs [#134936](https://github.com/elastic/elasticsearch/pull/134936)

> * [Transform] Wait for PIT to close [#134955](https://github.com/elastic/elasticsearch/pull/134955)

Reviewer: ```suggestion
* Fixes a bug where transforms did not wait for PITs to close [#134955](https://github.com/elastic/elasticsearch/pull/134955)
```

* Bypass MMap arena grouping as this has caused issues with too many regions being mapped [#135012](https://github.com/elastic/elasticsearch/pull/135012)
* Fixes deadlock in ThreadPoolMergeScheduler when a failing merge closes the IndexWriter [#134656](https://github.com/elastic/elasticsearch/pull/134656)

> * Fixes countDistinctWithConditions in csv-spec tests [#135097](https://github.com/elastic/elasticsearch/pull/135097)

Reviewer: ```suggestion
* Fixes `countDistinctWithConditions` in csv-spec tests [#135097](https://github.com/elastic/elasticsearch/pull/135097)
```

> * CentroidCalculator does not return negative summation weights [#135176](https://github.com/elastic/elasticsearch/pull/135176)

Reviewer: ```suggestion
* Fixes a bug where `CentroidCalculator` did not return negative summation weights [#135176](https://github.com/elastic/elasticsearch/pull/135176)
```

* ES|QL: Limit when we push topn to lucene [#134497](https://github.com/elastic/elasticsearch/pull/134497)
* Ban Limit + MvExpand before remote Enrich [#135051](https://github.com/elastic/elasticsearch/pull/135051)

> * Fix expiration time in ES|QL async [#135209](https://github.com/elastic/elasticsearch/pull/135209)

Reviewer: ```suggestion
* Fixes expiration time in {{esql}} async [#135209](https://github.com/elastic/elasticsearch/pull/135209)
```

* Adds a check to confirm that uploaded files are indexed and searchable in {{ml-cap}} [#231614]({{kib-pull}}231614)
* Updates sections and improves field handling in {{ml-cap}} [#231037]({{kib-pull}}231037)
* Improves the layout of the custom URLs list in {{ml-cap}} [#231751]({{kib-pull}}231751)

> * Return 429 instead of 500 for timeout handlers [#133111](https://github.com/elastic/elasticsearch/pull/133111)

Reviewer: ```suggestion
* Returns 429 status code instead of 500 for timeout handlers [#133111](https://github.com/elastic/elasticsearch/pull/133111)
```

> * Allow configuring SAML private attributes [#133154](https://github.com/elastic/elasticsearch/pull/133154)

Reviewer: ```suggestion
* Allows configuring SAML private attributes [#133154](https://github.com/elastic/elasticsearch/pull/133154)
```

* Adds inline markdown visualization [#229191]({{kib-pull}}229191)
* Adds an `AI` section to the `Stack Management` menu [#227289]({{kib-pull}}227289)
* Sets the default retention period for Logs anomaly detection to 120 days [#231080]({{kib-pull}}231080)
* Restrict Indexing To Child Streams When Streams Is Enabled [#132011](https://github.com/elastic/elasticsearch/pull/132011)

> * [ML] Adds support for dimensions in google vertex ai request [#132689](https://github.com/elastic/elasticsearch/pull/132689)

Reviewer: ```suggestion
* Adds support for passing the `dimensions` field in the Google Vertex AI request [#132689](https://github.com/elastic/elasticsearch/pull/132689)
```

* Improves rate limiter UX [#227678]({{kib-pull}}227678)
* Adds table list view to the space selector screen [#229046]({{kib-pull}}229046)
* Adds `kibana.alert.grouping` field to infra alerts [#229054]({{kib-pull}}229054)

> * Skip search shards with INDEX_REFRESH_BLOCK

Reviewer: ```suggestion
* Skips search shards with `INDEX_REFRESH_BLOCK`
```

> * Pipelines: Adds `created_date` and `modified_date` #130847](https://github.com/elastic/elasticsearch/pull/130847)

Reviewer: ```suggestion
* Adds the `created_date` and `modified_date` system-managed properties to pipelines #130847](https://github.com/elastic/elasticsearch/pull/130847)
```

* Component Templates: Adds `{created,modified}_date` [#131536](https://github.com/elastic/elasticsearch/pull/131536)

> * [Security] Adds entity store and asset criticality index privileges to built in roles [#129662](https://github.com/elastic/elasticsearch/pull/129662)

Reviewer: ```suggestion
* Adds entity store and asset criticality index privileges to built-in roles [#129662](https://github.com/elastic/elasticsearch/pull/129662)
```

* Removes metrics and logs from the `get_service_stats` API [#218346](https://github.com/elastic/kibana/pull/218346)
* Allows you to customize the table tab [#218686](https://github.com/elastic/kibana/pull/218686)
* Enables keyboard navigation for the create annotations form [#217918](https://github.com/elastic/kibana/pull/217918)

> * Updating tika to 2.9.3 [#127353](https://github.com/elastic/elasticsearch/pull/127353)

Reviewer: ```suggestion
* Updates tika to 2.9.3 [#127353](https://github.com/elastic/elasticsearch/pull/127353)
```
